extensions/applicationinsights-properties-js/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
is this one for amd? perhaps name the file tsconfigamd.json

Reply:
for amd. Will rename.

---
In reply to: [214087985](https://github.com/Microsoft/ApplicationInsights-JS/pull/660#discussion_r214087985) [](ancestors = 214087985)